pkgs/development/python-modules/torch/bin.nix

```diff
@@ -40,18 +40,19 @@ in buildPythonPackage {
 
   nativeBuildInputs = [
     addOpenGLRunpath
+    patchelf
```

Comment:
patchelf is already included in stdenv

Reply:
https://github.com/NixOS/nixpkgs/pull/236793